CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ endif(COMMAND cmake_policy)
 
 project(parameter-framework)
 
+# find and set the Parameter Framework's version
+execute_process(COMMAND git describe --tags --abbrev=0
```

Comment:
I would rather have the exact version, because (one patch over v1) != (v1). => remove --abbrev=0
It would also be good to know if the tree is dirty. => add --dirty

~~~
  git describe --tags --dirty
~~~